Library/Formula/plplot.rb

```diff
@@ -12,11 +12,14 @@ class Plplot < Formula
 
   option 'with-java'
 
+  patch :DATA
```

Comment:
I'm not a brew maintainer, so take this with a grain of salt, but I think they want a comment above each patch indicating why it is necessary.